### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by N-o-Z**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Current Behavior
+*What actually happened, including error codes if applies.*
+
+## Expected Behavior
+*How you expected thing to work.*
+
+## Logs
+<!--
+If available provide any logs relevant to the issue (lakeFS, clients, ecosystem)
```
Comment (arielshaqed) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]:
```suggestion
If available, please provide any logs relevant to the issue (lakeFS, clients, ecosystem). Make sure to scrub anything confidential! These will typically be near strings such as `secret`, `access` or `AKIA`
```

**Review by arielshaqed — COMMENTED** [review]
Neat! Not approving because this is a drive-by review.

Can we ask for (optional) contact details here? Should we???

**Comment by nopcoder** [comment]
@N-o-Z there is a new template mechanism in github that allow advance form capabilities, maybe worth checking out: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

**Comment by N-o-Z** [comment]
> @N-o-Z there is a new template mechanism in github that allow advance form capabilities, maybe worth checking out: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

This is exactly the guide I was using to create the template - the result is this PR

**Review by hiccups28 — COMMENTED** [review]
Looks good to me, shall we run a sample to test?

**Comment by N-o-Z** [comment]
@arielshaqed @hiccups28 @nopcoder  Please have another look - I've changed the format to use the forms beta feature.
You can see it properly rendered from the branch's code section

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  - type: input
+    id: deployment
+    attributes:
+      label: Deplyoment
+      description: lakeFS deployment type
+      placeholder: ex. AWS ECS, Docker or Kubernetes, helm-chart, local
+    validations:
+      required: true
+  - type: input
+    id: Clients
```
Comment (arielshaqed) [inline .github/ISSUE_TEMPLATE/bug-report.yml:54]:
```suggestion
    id: clients
```

**Review by arielshaqed — APPROVED** [review]
Yay, very neat!

Small request: Do we want to add _another_ box, for reporters to be able to say some more stuff that doesn't fit anywhere else?  That way the fields are what we know _helps_ us, but don't _force_ the reporter to do anything in particular.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      description: If applicable provide lakeFS client and version that was used
+      placeholder: ex. hadoop-lakefs 0.1.9, python lakefs-client 0.89.0
+    validations:
+      required: false
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant logs output
+      description: Please copy and paste any relevant log output (lakeFS, clients, ecosystem). Make sure to scrub anything confidential! These will typically be near strings such as `secret`, `access` or `AKIA`
+      render: shell
```
[the remark this answers is not in the document]
Comment (N-o-Z) [reply .github/ISSUE_TEMPLATE/bug-report.yml:66]: This is the recommendation rendering for log textarea from github - we can change it later if it doesn't render well

**Review by nopcoder — APPROVED** [review]
Very nice addition!
- Can we move the `contact` to the end
- Would keep only `what-happened` as required
- Optional: split the what and expected into different fields
- If possible remove `Code of Conduct` section

**Pull request merged**